main.py

```diff
@@ -1,15 +1,17 @@
 from modules import Clients, ServerWithGCI, GCI
 import simpy
 
-SIM_DURATION = 10
+SIM_DURATION = 12
```

Comment:
Is there unit associated with this duration? It would be clearer if you suffix the name with it. For example, if this is milliseconds, you would make this variable SIM_DURATION_MS.

Reply:
Alright!

Reply:
Done, @danielfireman could you review?

Reply:
On it.